src/hotspot/share/gc/g1/g1CardSet.cpp

```diff
@@ -192,7 +192,7 @@ class G1CardSetHashTable : public CHeapObj<mtGCCardSet> {
   };
 
   class G1CardSetHashTableFound : public StackObj {
-    G1CardSetHashTableValue* _value;
+    G1CardSetHashTableValue* _value = nullptr;
```

Comment:
Non-static data member initializers (http://www.open-std.org/JTC1/SC22/WG21/docs/papers/2008/n2756.htm) have not (yet) been approved for use in HotSpot.  Needs to be initialized in a constructor.